src/main/java/pl/vm/javaguild/designpatterns/pattern/creational/factorymethod/Department/Enums/Section.java

```diff
@@ -0,0 +1,9 @@
+package pl.vm.javaguild.designpatterns.pattern.creational.factorymethod.Department.Enums;
```

Comment:
osobiscie nie mam problemow z folderami "Enums" , ale znam architektow ktorzy tego nie toleruja - na przyszlosc mozesz uzywac czegos innego jak "Model" etc.
Logika jest taka, ze skoro mamy folder Enums do enum'ow to powinnismy miec foldery Interfaces do interface etc.

Reply:
Zgadzam się z @LukaszCaputa. Pakiet `model` to dobre miejsce dla enumów.